.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+* @holgiosalos @danielgalvis98 @kliver98 @AlejaGonzalezV @NicolasB2 @manuelq12 @Valeryibarra @veronicatofino
```

Comment:
Add a break line at the end to avoid this warning to github :)
![image](https://user-images.githubusercontent.com/34688953/176077858-5e492eba-8147-48a3-bebf-e82212630e22.png)
The end of all files should be always an empty line.

For future comments with this issue, we will only put (EOF) as reference.